File: 2942-find-words-containing-character/2942-find-words-containing-character.cpp
class Solution {
public:
    vector<int> findWordsContaining(vector<string>& words, char x) {
        vector<int> res;

        for(int i = 0; i<words.size(); i++){
            string currstr =  words[i];

            for(int j = 0; j<currstr.length(); j++){
                if(currstr[j] == x){
                    res.push_back(i);
                    break;
                }
            }
        }
        return res;
    }
};

File: 2942-find-words-containing-character/2942-find-words-containing-character.js
/**
 * @param {string[]} words
 * @param {character} x
 * @return {number[]}
 */
var findWordsContaining = function(words, x) {
    let ans = [];

    for(let i= 0; i<words.length; i++){
        let currstr = words[i];

        for(let j = 0; j<currstr.length; j++){
           if(currstr[j] === x){
            ans.push(i);
            break;
           }
        }
    }
    return ans
};